api/v1beta1/etcdadmcluster_types.go

```diff
@@ -26,6 +26,8 @@ import (
 const (
 	UpgradeInProgressAnnotation = "etcdcluster.cluster.x-k8s.io/upgrading"
 
+	PausedHealthCheckAnnotation = "etcdcluster.cluster.x-k8s.io/healthcheck-paused"
```

Comment:
What if we made this annotation something like `etcdcluster.cluster.x-k8s.io/healthcheck-retries`?
That would give users the option to configure it a little bit on top of being able to disable it. For example, a value of 0 could mean healthcheck disabled.

Reply:
done